OpenOversight/app/main/views.py

```diff
@@ -609,11 +611,11 @@ def redirect_display_tag(tag_id: int):
 
 @main.route("/tags/<int:tag_id>")
 def display_tag(tag_id: int):
-    try:
-        tag = db.session.get(Face, tag_id)
-        proper_path = serve_image(tag.original_image.filepath)
-    except NoResultFound:
+    tag = db.session.get(Face, tag_id)
+    if not tag:
         abort(HTTPStatus.NOT_FOUND)
```

Comment:
It does not error if you give it an invalid ID, that is validated by the `test_invalid_tag_id_view_tag` test function.